data/mods/Magiclysm/Spells/attunements/Shapeshifter.json

```diff
@@ -0,0 +1,33 @@
+[
+  {
+    "type": "SPELL",
+    "id": "shapeshifter_alkahest",
+    "name": "Alkahest",
+    "description": "Release the mana-saturated liquid out your ley lines and cover everything with bioactive solvent.  Sadly you are the only person that have resistance to his effect, so its better to be alone when you cast it.",
```

Comment:
this description needs a little bit of grammatical work.

Reply:
maybe something like 
"Release a spray of mana-saturated bioactive solvent. Your ley lines give you a resistance to this acid due to your mana signature, but others beware!"
I know it's a slightly different message so take the suggestion as you will

Reply:
no problem, in original message i said it must be rewrited, cause im not the best in description stuff